.github/workflows/_run_android_tests.yml

```diff
@@ -61,10 +45,10 @@ jobs:
           ANDROID_ROOT="/usr/local/lib/android"
           ANDROID_SDK_ROOT="${ANDROID_ROOT}/sdk"
           SDKMANAGER="${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager"
-          echo "y" | $SDKMANAGER "ndk;21.4.7075529"
+          echo "y" | ${SDKMANAGER} "ndk;21.4.7075529"
```

Comment:
Why curly braces around single variable here and below?

Reply:
Somewhat I find it in general more readable, there is some level of discussion on the topic, but in general I see it being in favour of preferring the braces (such as https://bertvv.github.io/cheat-sheets/Bash.html)